www/directives/player/mission.html

```diff
@@ -1,37 +1,46 @@
 <div>
-	<div class='panel panel-danger' ng-if='missionCtrl.missionStatus.status == "locked"'>
-		<div class='panel-body'>
-			<p class='lead text-danger'>Mission locked</p>
-		</div>
+
+	<!-- Mission is locked -->
+	<div class='container' ng-if='missionCtrl.missionStatus.is_locked'>
+		<mission-locked />
 	</div>
-	<div class='col-xs-12'>
-		<div class='panel panel-default'>
-			<div class='panel-body'>
-				<h3>{{missionCtrl.missionStatus.mission.title}}</h3>
 
-				<table class='table pull-right' width='300px'>
-					<tr>
-						<th>Status</th>
-						<th>
-							<span ng-if='missionCtrl.missionStatus.is_locked' class='glyphicon glyphicon-lock text-muted' />
-							<span ng-if='missionCtrl.missionStatus.is_open' class='glyphicon glyphicon-remove text-warning' />
-							<span ng-if='missionCtrl.missionStatus.is_success' class='glyphicon glyphicon-ok text-success' />
-							{{missionCtrl.missionStatus.status}}
-						</th>
-						<th>+ {{missionCtrl.missionStatus.mission.reward}} pts</th>
-					</tr>
-					<tr ng-repeat='star in missionCtrl.missionStatus.mission.stars.__items'>
-						<th width='15%'>
-							<span class='glyphicon'
-								ng-class="{ 'glyphicon-star-empty text-warning': !missionCtrl.statusByStar(star._id), 'glyphicon-star text-success': missionCtrl.statusByStar(star._id) }" />
-						</th>
-						<td width='15%'>{{star.reward}} pts</td>
-						<td>{{star.title}}</td>
-					</tr>
-				</table>
-				<div ng-bind-html="missionCtrl.missionStatus.mission.desc"></div>
+	<!-- Mission not locked -->
+	<div ng-if='!missionCtrl.missionStatus.is_locked'>
```

Comment:
Please, can I still have the mission even if not allowed, to simplify the debug? See #77

Anyway, it's not the role of the angular to hide information, it's the role of the server to not send it.